patches/tools-gritsettings-resource_ids.spec.patch

```diff
@@ -12,7 +12,12 @@ index 2450f2cece03374b6ee91b5e7b4e4f112ecefbb3..98b058575f95c52dce48be4ae50633b6
    "chrome/app/google_chrome_strings.grd": {
      "messages": [800],
    },
-@@ -300,7 +303,7 @@
+@@ -296,11 +299,11 @@
+     "includes": [3280],
+   },
+   "<(SHARED_INTERMEDIATE_DIR)/chrome/browser/resources/extensions/resources.grd": {
+-    "META": {"sizes": {"includes": [90],}},
++    "META": {"sizes": {"includes": [91],}},
```

Comment:
why do we need to patch this?

Reply:
If we don't patch, we get
```grit.exception.IdRangeOverflow: ID range overflow.: Generated .grd file used more IDs (91) than were allocated for it (90) for type includes. You need to update ..\..\..\..\..\gen\tools\gritsettings\default_resource_ids for this .grd.```

Reply:
I think that's because we add extra items to the grd with brave/patches/chrome-browser-resources-extensions-BUILD.gn.patch

Reply:
I wonder if there's some better way to do that, can we open a follow-up issue for it?

Reply:
Created https://github.com/brave/brave-browser/issues/40294